### ISSUE TITLE
Class 'Drupal\Sniffs\Semantics\FunctionCall' not found

### ISSUE THREAD

**Issue by pfrenssen** [pfrenssen/coder#14]
I'm getting the following error when trying to run the coding standards checks with Coder 8.3.0 and PHP_CodeSniffer 3.3.1:

> PHP Fatal error:  Class 'Drupal\Sniffs\Semantics\FunctionCall' not found in /home/pieter/v/joinup/vendor/drupal/coder/coder_sniffer/Drupal/Sniffs/Semantics/FunctionTSniff.php on line 24

**Comment by fenetikm** [comment]
Thanks, will take a look on the weekend.

**Comment by pfrenssen** [comment]
After some investigation it appears that this only occurs when referring to the Drupal standard in a custom ruleset. When using the unmodified Drupal standard directly the error does not occur. It can be replicated by using the following `phpcs-ruleset.xml` file:

```
<?xml version="1.0"?>
<ruleset name="Drupal">
    <rule ref="./path/to/drupal/coding/standard" />
</ruleset>
```

**Comment by stefanpl** [comment]
I'm still seeing this issue with the exact usecase you described – referencing the Drupal standard in a custom xml file (`coder 8.3.0`, `codesniffer 3.5.2`). Did you find a solution here?

**Comment by pfrenssen** [comment]
Yes as you can see in the linked PR this was fixed in Coder 8.3.1. It should be fixed if you upgrade to the latest version of Coder. If not, feel free to open a new issue.
